SU-SOE/stanford_soe_helper
# mobile mag style fixes

# READY FOR REVIEW

# Summary
- Made requested tweaks to the mobile styles for magazine articles

# Needed By (Date)
- End of sprint - 11/2

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that styles are rendering as they should (see ticket linked to below for more details, if needed). Most obvious fix should be that the text no longer runs off of the page.

# Affected Projects or Products
- Enginnering
- stanford_soe_helper

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-3670

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
